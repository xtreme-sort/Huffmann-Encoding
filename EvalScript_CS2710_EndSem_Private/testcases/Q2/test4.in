DeriveHuffmanTree
19
0 1101
3 010111
4 111011
7 1010
A 1110100
C 001
K 01010
L 1100
R 11100
U 100
a 01110
b 1110101
g 010110
h 1011
k 1111
l 0100
m 000
p 01111
q 0110
Quit
