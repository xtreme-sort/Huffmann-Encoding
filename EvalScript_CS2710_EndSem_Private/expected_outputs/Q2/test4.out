(((m,C),((l,(K,(g,3))),(q,(a,p)))),((U,(7,h)),((L,0),((R,((A,b),4)),k))));
